src/status_im/transport/inbox.cljs

```diff
@@ -179,6 +180,13 @@
 (def ^:private ^:const max-retries 10)
 (def ^:private ^:const retries-interval-change-threshold 3)
 
+(defn add-custom-mailservers [mailservers {:keys [db]}]
+  {:db (reduce (fn [db {:keys [id chain] :as mailserver}]
+                 (assoc-in db [:inbox/wnodes (keyword chain) id]
+                           (-> mailserver
+                               (dissoc :chain)
+                               (assoc :user-defined true)))) db mailservers)})
```

Comment:
I would prefer `reduce` arguments to be vertically align https://github.com/bbatsov/clojure-style-guide#vertically-align-fn-args
```clj
(reduce (fn [] ...
        db
        mailservers)
```